### ISSUE TITLE
Test with real FITS in Travis

### ISSUE THREAD

**Issue by awead** [psu-libraries/cho#591]
We're currently mocking calls to FITS when testing in Travis, although local testing will use FITS for real. We could configure Travis to download FITS and install it. Ideally, the download could be cached so as not to impact testing time. Or, we could just choose to continue to mock it.

**Comment by rschenk** [comment]
I hacked at this for a couple hours but didn't make any real progress. Here's what I found:

Someone made a FITS package for debian: 
https://github.com/ConorSheehan1/fits.deb
https://launchpad.net/~conorsheehan1/+archive/ubuntu/fits-ppa

I added the following to our travis.yml file:
```
addons:
  apt:
    sources:
      - sourceline: 'ppa:conorsheehan1/fits-ppa'
    packages:
      - fits
```

Travis does attempt to install the package, but fails:
https://travis-ci.com/psu-libraries/cho/jobs/200455059

It's kind of hard to parse out the error messages, but in a nutshell it seems that the fits-ppa package does not support the platform/os/vm that Travis is running on?

**Comment by awead** [comment]
I think we should just continue to mock FITS in Travis or future CI tools. We would notice changes to its output during QA testing, and testing of FITS as it integrates into the overall framework might be better subject of our future deployment strategies with containers, docker, kubernetes, and the like.
